pkg/stacks/relationship.go

```diff
@@ -33,9 +36,19 @@ const (
 	LabelParentName      = "core.crossplane.io/parent-name"
 	LabelParentUID       = "core.crossplane.io/parent-uid"
 
-	LabelMultiParentPrefix   = "parent.stacks.crossplane.io/"
+	LabelMultiParentPrefix = "parent.stacks.crossplane.io/"
+
 	LabelMultiParentNSFormat = "parent.stacks.crossplane.io/%s"
-	LabelMultiParentFormat   = LabelMultiParentNSFormat + "-%s"
+
+	// LabelMultiParentFormat defines the format for combining a
+	// LabelMultiParentNSFormat with a named resource
+	LabelMultiParentFormat = "%s-%s"
+
+	// preserveNSLength is the number of characters using the label name that
+	// will be dedicated to identifying the namespace. This length will include
+	// truncation characters if the namespace exceeds this length.
+	// example: parent.stacks.crossplane.io/{up to 32 chars of NS}-{Name}
```

Comment:
Excellent detailed comment with example. This helps me reason about the intended behavior of the implementation. :squid: